src/content/guides/build-performance.md

```diff
@@ -115,6 +115,9 @@ The following utilities improve performance by compiling and serving assets in m
 - `webpack-hot-middleware`
 - `webpack-dev-middleware`
 
+### stats.toJson speed
+
+webpack4 outputs a large amount of data with its `stats.toJson()` by default. Avoid retrieving portions of the `stats` object unless absolutely necessary in the incremental step. `webpack-dev-server` after v3.1.3 contained a substantial performance fix to minimize the amount of data retrieved from the `stats` object per incremental build step.
```

Comment:
Please add a space between webpack and its version:
> webpack 4

Reply:
Finally got some time to fix this one :)